### PR TITLE
Assorted Fixes ahead of professor demo

### PR DESCRIPTION
**Pull request by Glitched**
* Change professor view colors. 
* Fix a minor render bug in Safari
* Get tag level from server & use SelectedTag component

### CODE REVIEW

**Comment by karunsingh** [comment]
(Just so that I don't forget, we should add the HTTPS redirect to this PR)

File: server/index.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ import { makeExecutableSchema, addMockFunctionsToSchema } from 'graphql-tools';
 import { graphql } from 'graphql';
 import { readFileSync, readFile } from 'fs'
 
+const { Client } = require('pg')
```
Comment (karunsingh) [inline server/index.ts]: Should we leave these changes for another PR?

**Comment by karunsingh** [comment]
Just noticed that the desktop design requires us to highlight the selected office hour in the sidebar: https://app.zeplin.io/project/5a0ccb7c3ed1f8ae9afa0adf/screen/5aeb4dbc39101c527972d1d9

File: client/src/components/pages/SplitView.tsx [hunk 2]
```diff
@@ -184,6 +184,7 @@ class SplitView extends React.Component {
                                     sessionId={this.state.sessionId}
                                     data={{}}
                                     callback={this.handleBackClick}
+                                    isDesktop={this.state.width > 800}
```
[the remark this answers is not in the document]
Comment (karunsingh) [reply client/src/components/pages/SplitView.tsx]: I think we can get away with a global constant without that:

https://stackoverflow.com/questions/39036457/react-create-constants-file

**Comment by Glitched** [comment]
For selected tags, do you think we could ask the designers to unify desktop/phone or pick one? 

https://app.zeplin.io/project/5a0ccb7c3ed1f8ae9afa0adf/screen/5abd3fdfbdf94bbd4ab27a19 doesn't have borders

There's a bunch of other small things that need done, too, like replacing the chevrons, restyling the week picker on the prof view and more - this commit isn't exhaustive yet, but just all the time I had to devote to it yesterday

File: client/src/components/includes/ProfessorSidebar.tsx [hunk 3]
```diff
@@ -52,11 +51,11 @@ class ProfessorSidebar extends React.Component {
                     </div>
                 </div>
                 <svg className="logo" width="100" height="100">
```
[the remark this answers is not in the document]
Comment (Glitched) [reply client/src/components/includes/ProfessorSidebar.tsx]: Do you know if they are the same dimensions, as in can we do a drop in replacement? It might be a good idea to change this PR to aim for near-full design parity

File: client/src/components/pages/SplitView.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Glitched) [reply client/src/components/pages/SplitView.tsx]: Is there a way we can access those constants in our LESS stylesheets, too? I use them for CSS media queries in addition to standard conditional rendering

**Comment by karunsingh** [comment]
(Continued in #54)
